addon/components/ce/content-editable.js

```diff
@@ -318,19 +322,51 @@ export default class ContentEditable extends Component {
     }
   }
 
-  /**
-   * cut isn't allowed at the moment
-   */
   @action
   cut(event) {
     event.preventDefault();
+
```

Comment:
I'd consider moving this to it's own inputhandler similar to other handlers (though admittedly paste does not have a corresponding handler either atm)